addons/project/static/src/views/project_task_form/project_task_form_controller.js

```diff
@@ -44,10 +44,13 @@ export class ProjectTaskFormController extends FormController {
             return;
         }
 
+        if (!record.resId) {
+             return;
+        }
```

Comment:
I'm not sure that's possible. I think that by default, when you click on the history button, it saves your record, but maybe I'm wrong?

Reply:
So you mean that this part is not necessary ? i'm not sure too :rofl: